www/_posts/2017-12-20-plugins-browser-release.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+author:
+    name: Steve Gill
+    url: https://twitter.com/stevesgill
+title:  "Plugins & Cordova-Browser Release"
+categories: news
+tags: release plugins
+---
+
+The following plugins were updated today:
+
+* cordova-plugin-battery-status@2.0.0
+* cordova-plugin-camera@4.0.0
+* cordova-plugin-contacts@3.0.1
+* cordova-plugin-device@2.0.0
+* cordova-plugin-dialogs@2.0.0
+* cordova-plugin-file@6.0.0
+* cordova-plugin-geolocation@4.0.0
+* cordova-plugin-globalization@1.0.9
+* cordova-plugin-inappbrowser@2.0.0
+* cordova-plugin-media@5.0.0
+* cordova-plugin-media-capture@3.0.0
+* cordova-plugin-network-information@2.0.0
+* cordova-plugin-splashscreen@5.0.0
+* cordova-plugin-statusbar@2.4.0
+* cordova-plugin-screen-orientation@3.0.0
+* cordova-plugin-vibration@3.0.0
+
+`cordova-browser` was also updated today to `5.0.2`    
+
+In the recent `cordova@8` tools release, we dropped support for deprecated platforms (Ubuntu, BlackBerry10, Windows Phone 8, FirefoxOS). The main focus of this plugins release was to drop support for those platforms as well. That is why many of the plugins have had major version increases. 
+
+`cordova-plugin-contacts` and `cordova-plugin-globalization` have officially been deprecated. Read about our decision to deprecate `cordova-plugin-contacts` at https://cordova.apache.org/news/2017/11/27/Deprecation-of-cordova-contacts-plugin.html. For `cordova-plugin-globalization`, we have written a migration guide over to the built-in [ECMA Internationalization API](https://www.ecma-international.org/ecma-402/1.0/) at https://cordova.apache.org/news/2017/11/20/migrate-from-cordova-globalization-plugin.html.
+----
```

Comment:
Need a line of whitespace above this separator, otherwise the paragraph above turns into a heading